core/validators/witness.go

```diff
@@ -345,8 +338,6 @@ func (w *Witness) OnTick(ctx context.Context, t time.Time) {
 				}
 				w.log.Warn("resource checking was not validated by all nodes",
 					logging.String("resource-id", v.res.GetID()),
-					logging.Int("vote-count", votesLen),
```

Comment:
Should we actually keep some kind of logs to show how much of the total we have?

Reply:
added